src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.LeadingZeroCount.cs

```diff
@@ -29,18 +29,52 @@ public static void LeadingZeroCount<T>(ReadOnlySpan<T> x, Span<T> destination)
         internal readonly unsafe struct LeadingZeroCountOperator<T> : IUnaryOperator<T, T> where T : IBinaryInteger<T>
         {
             public static bool Vectorizable =>
-                (Avx512CD.VL.IsSupported && (sizeof(T) == 4 || sizeof(T) == 8)) ||
+                (Avx512CD.VL.IsSupported && (sizeof(T) == 2 || sizeof(T) == 4 || sizeof(T) == 8)) ||
+                (Avx512BW.IsSupported && Avx512Vbmi.IsSupported && Avx512Vbmi.VL.IsSupported && sizeof(T) == 1) ||
```

Comment:
This can just be
```suggestion
                (Avx512Vbmi.VL.IsSupported && sizeof(T) == 1) ||
```

`Avx512BW` and `Avx512Vbmi` are implied by `Avx512Vbmi.VL`